extensions/eks/main.tf

```diff
@@ -55,7 +55,7 @@ module "eks" {
   source          = "terraform-aws-modules/eks/aws"
   version         = "18.26.5"
   cluster_name    = local.cluster_name
-  cluster_version = "1.21"
+  cluster_version = "1.22"
```

Comment:
I snuck this in cause I was getting failures from AWS trying to deploy 1.21
https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html